src/API/Helpers/State/SessionStateHandler.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Auth0\SDK\API\Helpers\State;
+
+use Auth0\SDK\Store\SessionStore;
+use Auth0\SDK\Exception\CoreException;
+
+/*
+ * This file is part of Auth0-PHP package.
+ *
+ * (c) Auth0
+ *
+ * For the full copyright and license information, please view the LICENSE file
+ * that was distributed with this source code.
+ */
+
+/**
+ * This interface must be implemented by state handlers.
```

Comment:
very minor, but these were copied on all implementations, copy/pasting from the interface :).

Might be confusing to read this, since these are the implementations themselves.